packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/ImportTransactionModal/InputCSV.tsx

```diff
@@ -0,0 +1,38 @@
+import { useRef } from 'react';
+import styled from 'styled-components';
+import { Textarea, Button } from '@trezor/components';
+
+import { Translation } from 'src/components/suite';
+
+const Wrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    min-height: 250px;
+    align-items: center;
+`;
+
+const StyledTextArea = styled(Textarea)`
+    flex: 1;
+`;
```

Comment:
Same result, but a bit cleaner:

```suggestion
const Wrapper = styled.div`
    display: flex;
    flex-direction: column;
    min-height: 250px;
    justify-content: space-between;
    gap: 20px;
`;
```

Reply:
cool, done in [338cca7](https://github.com/trezor/trezor-suite/pull/10331/commits/338cca7524d208c082812c6d24c5b67e90e66944)